pkg/deployment/resources/pod_creator.go

```diff
@@ -85,8 +96,17 @@ func createArangodArgs(apiObject metav1.Object, deplSpec api.DeploymentSpec, gro
 		// With authentication
 		options = append(options,
 			optionPair{"--server.authentication", "true"},
-			optionPair{"--server.jwt-secret", "$(" + constants.EnvArangodJWTSecret + ")"},
 		)
+		if versionHasJWTSecretKeyfile(version) {
+			keyPath := filepath.Join(k8sutil.ClusterJWTSecretVolumeName, constants.SecretKeyToken)
```

Comment:
Don't think using the volume name is correct.
Should be the mount path of that volume (`ClusterJWTSecretVolumeMountDir`)

Reply:
Right.